index.html

```diff
@@ -127,7 +127,7 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
           <li>CSS</li>
           <li>Ruby</li>
         </ul>
-        <button type="button" class="see-project-card" id="button-2">
+        <button type="button" class="see-project-card" id="button-2" onclick="openDetail(2)">
```

Comment:
- [ ] Kindly ensure your design matches the given design, currently, the layout is a bit different on the pop-up window, also one of your popups doesn't show the projected image.


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/169467575-2961f524-378f-439e-b560-9f0ac8374c77.png) |  ![image](https://user-images.githubusercontent.com/61408860/169467621-1082e462-cdfc-484e-a1c8-141c16581fc1.png)


- [ ] Kindly also check your design at different widths, also the page doesn't fit in the entire window, below is a video showing it, I have indicated the design difference in mobile view compared to the given design kindly fix it.[Click here](https://www.loom.com/share/d43f0fff0f92488282fc4ca000e592e1) to view the video.